### PR TITLE
chore(website): fixes three documentation issues

### PR DESCRIPTION
**Pull request by TheSisb**
Fixes:
 - [x] https://github.com/twilio-labs/paste/issues/1141 Tooltip requires forwardRef on child but not documented
 - [x] https://github.com/twilio-labs/paste/issues/814 Maintainers dead link in readme file
 - [x] https://github.com/twilio-labs/paste/issues/451 DoDont lack of clarity on icon usage page

### CODE REVIEW

File: packages/paste-website/src/pages/icons/usage-guidelines/index.mdx [hunk 1]
```diff
@@ -43,9 +43,9 @@ export const pageQuery = graphql`
 
 <Callout>
   <CalloutText>
-      Our icon set is still limited, and more will be added over time. Please <Anchor href="https://github.com/twilio-labs/paste/issues">
-      file an icon request
-    </Anchor> or follow the guide on <SiteLink to="/icons/how-to-add-an-icon">how to add an icon</SiteLink>.
+    Our icon set is still limited, and more will be added over time. Please{' '}
+    <Anchor href="https://github.com/twilio-labs/paste/issues">file an icon request</Anchor> or follow the guide on{' '}
```
Comment (katieporter) [inline packages/paste-website/src/pages/icons/usage-guidelines/index.mdx:47]: Should this link to Discussions now instead of Issues?

Comment (richbachman) [reply packages/paste-website/src/pages/icons/usage-guidelines/index.mdx:47]: 👍 I feel like thats where we get the requests.

**Review by richbachman — APPROVED** [review]
Other than the comment about icon requests, this lgtm.

File: packages/paste-website/src/pages/components/tooltip/index.mdx [hunk 2]
```diff
@@ -82,22 +82,28 @@ the keyboard or on mouse hover. Tooltip follows the
...
 
-#### Accessibility
+### Accessibility
 
 A tooltip **must only** be placed on a **natively focusable** HTML element. Good candidates include a [Paste Button](/components/button) or a [Paste Anchor](/components/anchor) if the tip also links to a help article.
 
 **Do not** place tooltips on non-focusable elements, like an icon.
 
+If your tooltip wraps a natively focusable HTML element that includes only an icon, make sure you set the `decorative` prop to false, and
+give the icon a title. The title of the icon should describe what the icon is representing, and the tooltip provides the additional context.
```
Comment (SiTaggart) [inline packages/paste-website/src/pages/components/tooltip/index.mdx:104]: We could probably be clearer here. 

To provide an example, if you have a delete phone number button who's only child is a trash can icon, the title of the icon should be the accessible name of the button action. I'd want it to say "delete phone number", not "trash can".

The title of the icon should describe the action the native element will perform, not the visual representation of the icon

File: packages/paste-website/src/pages/icons/usage-guidelines/index.mdx (hunk 1 above)
Comment (SiTaggart) [reply packages/paste-website/src/pages/icons/usage-guidelines/index.mdx:47]: Considering we have quite a lot of icons now, should this change to "can't find the icon you need? Check streamline and contribute it to the library following our guidelines."

File: packages/paste-website/src/pages/components/tooltip/index.mdx [hunk 3]
```diff
@@ -155,19 +161,24 @@ A tooltip **must** be placed on a **focusable** element, such as an anchor or bu
...
-  <CalloutTitle>Icons and tooltips</CalloutTitle>
-  <CalloutText>
-    If your tooltip wraps an element that includes only an icon, make sure you set the `decorative` prop to false, and
-    give the icon a title. The title of the icon should describe what the icon is representing, and the tooltip
-    describes what that means. This ensures the icon and tooltip ara accessible to screen readers.
-  </CalloutText>
-</Callout>
+### Adding a Tooltip to non-Paste React components
+
+If you want to add a Tooltip around a React component that isn't in the Paste library, you must make sure that component [forwards refs](https://reactjs.org/docs/forwarding-refs.html).
```
Comment (SiTaggart) [inline packages/paste-website/src/pages/components/tooltip/index.mdx:166]: Not sure we need to mention non-paste components here. This is equally relevant if you wrap a paste component, but don't forward refs to it.

Maybe it's your teams "Buy" button that is used everywhere, and you create a BuyButton component that composes our Button into primary and full width state. That would need a ref forwarded to it

Comment (TheSisb) [reply packages/paste-website/src/pages/components/tooltip/index.mdx:166]: I had a hard time framing this in any other way that would be clear. Paste components `forwardRef` by default, so I wanted to describe what to look out for without having to explain/overwhelm by mentioning `forwardRef` in the heading.

Comment (SiTaggart) [reply packages/paste-website/src/pages/components/tooltip/index.mdx:166]: Yeah, it's not easy. I just don't feel like we should be documenting non-paste components outside of primitive pages.

Comment (TheSisb) [reply packages/paste-website/src/pages/components/tooltip/index.mdx]: Improved. Lmk!

**Pull request merged**